src/main/java/org/opentripplanner/standalone/config/NodeAdapter.java

```diff
@@ -492,4 +493,21 @@ else if(requireAllValues) {
         }
         return result;
     }
+
+    public Set<String> asStringSet(
```

Comment:
I created a very similar method called `asTextSet` for another PR. I think it can be used instead of this, isn't it?

Reply:
I called it `asTextSet` to be in line with other methods returning strings.

Reply:
I've updated the code to use `asTextSet()` :+1: